aliases/start-work/README.md

```diff
@@ -36,8 +41,11 @@ gh start-work <JIRA_TICKET> [ISSUE_TYPE]
 # Start work on a feature
 gh start-work PROJ-123
 
-# Start work on a bug fix
-gh start-work PROJ-456 fix
+# Start work on a feature with just a PR title (uses default feat type)
+gh start-work PROJ-123 "Add login functionality"
```

Comment:
Unrelated to this PR, but could we potentially shorten `start-work` to eg. `start` ?

Reply:
Will create a separate PR.